guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -48,7 +48,7 @@ ifdef::satellite[]
 # subscription-manager repos --enable {RepoRHEL8ServerSatelliteUtils}
 endif::[]
 # {foreman-maintain} packages unlock
-# yum install install python-pulp3-manifest -y
+# {package-install} python-pulp3-manifest
```

Comment:
```suggestion
# {package-install} python-pulp3-manifest -y
```

Reply:
Sorry, I didn't notice `-y` option removed in the previous review. @maximiliankolb is including `-y` for automatic installation not desirable?

Reply:
IMHO we rarely do that, so I'd rather say "no, not desirable".

Reply:
Alright. Sagar, please ignore this review.

Reply:
```suggestion
# {package-install} python3-pulp_manifest
```